### PR TITLE
made a change to add weather related information

### PR DESCRIPTION
**Pull request by samkrishnan**
This was CR # 420. This should save the world once again.

### CODE REVIEW

**Comment by NNSU99** [comment]
I have reviewed it...Thx!

**Review by NNSU99 — APPROVED** [review]

File: _slides/09-samkrishnan.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
----
-layout: slide
-title: "Welcome to our slide deck!"
----
-
-Caption here
-
+﻿---
+layout: slide
```
Comment (DekalbIrina) [inline _slides/09-samkrishnan.md:2]: This is a prove we can approve Pool Request

**Review by DekalbIrina — APPROVED** [review]
This is a FINISH reviews for TEST Purposes. Irina

**Comment by DekalbIrina** [comment]
Irina test for Review Pool Request.

**Comment by c4llmeco4ch** [comment]
Wow :+1:

**Review by NNSU99 — APPROVED** [review]

**Review by sammyseltzer — APPROVED** [review]
Your stuff looks great

File: _slides/09-samkrishnan.md (hunk 1 above)
Comment (NNSU99) [reply _slides/09-samkrishnan.md:2]: reviewed and approved

**Comment by NNSU99** [comment]
@samkrishnan @DekalbIrina ---good

**Review by NNSU99 — APPROVED** [review]

**Pull request merged**